lib/kube/proxy/forwarder.go

```diff
@@ -748,6 +819,24 @@ func (f *Forwarder) getKubeAccessDetails(
 	}, nil
 }
 
+// podRegex is the Pods endpoint API url.
+var podRegex = regexp.MustCompile(`(?m)/api/v1/namespaces/([^/]+)/pods/([^/]+)`)
```

Comment:
Is the `(?m)` necessary?

Should the regex end in `$`?

Maybe:

```suggestion
var podNameRegex = regexp.MustCompile(`/api/v1/namespaces/([^/]+)/pods/([^/]+)$`)
```

Reply:
It can't end with an `$` since Kubernetes API has other endpoints such as `exec`, `portforward`